lib/baldr/validator.rb

```diff
@@ -24,10 +26,11 @@ def validate_tree!(segment, grammar, structure)
     l = 0
     structure.fetch(:level, []).each do |s|
       loop = segment.children[l]
+
       if loop && loop.id.to_s == s[:id]
-        check_loop_count(loop, s)
+        check_loop_count(loop, s, @truncate_loops)
```

Comment:
@scottkf _A now? 790edf3d6989854b322b8641d1b1e1807ade918e

The reason I'm passing around the boolean is explain in the commit msg.